integreat_cms/xliff/xliff1_serializer.py

```diff
@@ -54,6 +55,23 @@ def start_object(self, obj):
                 "target-language": obj.language.slug,
             },
         )
+        # This header is required to make sure the XLIFF file can be segmented with MemoQ's WPML filter to get the same
+        # translation memory as with the legacy export via WordPress/WPML. See also:
+        # https://docs.memoq.com/current/en/Places/wpml-xliff-filter.html
+        self.xml.startElement("header", {})
+        self.xml.startElement("phase-group", {})
+        self.xml.addQuickElement(
+            "phase",
+            attrs={
+                "phase-name": "shortcodes",
+                "process-name": "Shortcodes identification",
+            },
+        )
+        self.xml.addQuickElement(
+            "phase", attrs={"phase-name": "post_type", "process-name": "Post type"}
+        )
+        self.xml.endElement("phase-group")
+        self.xml.endElement("header")
```

Comment:
This produces the following output:
```
<header>
    <phase-group>
        <phase phase-name="shortcodes" process-name="Shortcodes identification"/>
        <phase phase-name="post_type" process-name="Post type"/>
    </phase-group>
</header>
```
If re really need both the open and close tags as stated in #1134, I think we have to insert a dummy note into these phases, otherwise the XML parser will always use `<phase/>` instead of `<phase></phase`.

Reply:
Maybe we can just try it out on our system and Vijay can check whether it works or not?

Reply:
Ok, sounds good :+1: